routes/event.js

```diff
@@ -55,6 +56,48 @@ router.get('/guests/:event_id',isLoggedIn, isAdminUser, function(req, res) {
 	});
 });
 
+router.get('/guests/send/:event_id', isLoggedIn, isAdminUser, function(req, res) {
+	eventMapper.findEventBy_event_id(req.params.event_id, function(err, result) {
+		if (err) {
+			res.send(err);
+		}
+
+		res.render('sendGuestDetails', {result, err: req.flash('err'),succ: req.flash('succ')});
+	});
+});
+
+router.post('/guests/send/:event_id', isLoggedIn, isAdminUser, function(req, res) {
+	eventMapper.findEventBy_event_id(req.params.event_id, function(err, result) {
+		if (!result) {
+			req.flash('err', 'Event not found');
+		}
+		else if (err) {
+			res.send(err);
+		}
+		else
+		{
+			var details = 'The following is a specification of guest dietary restrictions, access requirements and table arrangements for the upcoming event, ' + result.title + ' on ' + result.date + ':\n\n';
+
+			addGuestDetails(result, details, function(guestDetails) {
+				addTableDetails(result, guestDetails, function(guestAndTableDetails) {	
+					sendEmailToCatering(req.body.cateringEmail, guestAndTableDetails, function(error, information) {	
+						if (error) {
+							req.flash('error', error);
+						}
+						else
+						{
+							req.flash('information', information);
```

Comment:
I dont know anywhere where 'error',  'information' flash messages are displayed. We've used 'err' and 'succ' everywhere else. information returned by the email will be just be true or something so might want to hardcode a success message there